.gitattributes

```diff
@@ -0,0 +1 @@
+*.strings diff
```

Comment:
Run scripts/convert_string_files.sh to convert the files back to plain text format from the binary format that `tx` insists on.

Reply:
Thanks, it helped with build errors too.